Cargo.toml

```
@@ -1,23 +1,21 @@
 [package]
 name = "pio"
-version = "0.2.1"
+version = "0.3.0"
 authors = ["snek", "The rp-rs developers"]
-edition = "2018"
+edition = "2021"
 resolver = "2"
 license = "MIT"
 description = "Support for the Raspberry Silicon RP2040's PIO State Machines."
 repository = "https://github.com/rp-rs/pio-rs"
 
 [workspace]
-members = ["pio-proc", "pio-parser"]
+members = ["pio-proc", "pio-parser", "pio-core"]
 
 [dependencies]
-arrayvec = { version = "0.7", default-features = false }
-paste = "1.0"
-num_enum = { version = "0.7", default-features = false }
+pio-core = { path = "./pio-core" }
+pio-proc = { path = "./pio-proc" }
```

Need version numbers here to be able to publish this crate.
```suggestion
pio-core = { version = "0.3.0", path = "./pio-core" }
pio-proc = { version = "0.3.0", path = "./pio-proc" }
```

---

thanks! fixed.